content/administration/maintain/google_oauth.rst

```diff
@@ -0,0 +1,208 @@
+   :align: center
+   :class: img-fluid
+
+A window from **Google** will appear to complete the authorization process. Select the appropriate
+email address that is being configured in Odoo.
+
+.. image:: google_oauth/connect-account.png
+   :align: center
+   :class: img-fluid
+
+Click *Continue* to allow the exception and proceed to the next step.
+
+.. image:: google_oauth/google-unverified.png
+   :align: center
+   :class: img-fluid
+
+.. note::
+   This step will only appear if *No Organization* was chosen for the *Location* when
+   creating the project on the OAuth consent screen. This indicates a *Personal Gmail Account*
+   rather than *Google Workspace*.
+
+Allow Odoo to access the Google Account by clicking on *Continue* or *Allow*.
+
+.. image:: google_oauth/google-access-account.png
+   :align: center
+   :class: img-fluid
+
+The configuration will automatically load the token in Odoo, and a graphic stating *Gmail Token
+Valid* will appear in green.
+
+.. image:: google_oauth/gmail-token-valid.png
+   :align: center
+   :class: img-fluid
+
+Save the settings and *Test the Connection*.
+
+The Google Gmail account is now functioning in Odoo using OAuth to authenticate. The database is
+now sending safe, secure emails through Google using Oauth authentication.
+
+Google OAuth FAQ
+================
+
+**Production** VS **Testing** *Publishing Status*
+-------------------------------------------------
+
+Choosing *Production* as the *Publishing Status* (instead of *Testing*) will display the following
+warning message:
+
+.. warning::
+   OAuth is limited to 100 sensitive scope logins until the OAuth consent screen is verified. This
+   may require a verification process that can take several days.
+
+To correct this warning, navigate to the `Google APIs platform
+<https://console.cloud.google.com/apis/credentials/consent>`__. If in *Production*, then click
+*Back to Testing* to correct the issue.
+
+.. image:: google_oauth/back-to-testing.png
+   :align: center
+   :class: img-fluid
+
+**Desktop App** VS **Web Application** *Application Type*
+---------------------------------------------------------
+
+Should *Desktop App* under the *Application Type* be selected when creating the *Credentials (OAuth
+client ID)*, the following error message will display:
+
+.. image:: google_oauth/error-400.png
+   :align: center
+   :class: img-fluid
+
+To correct this error, delete the credentials already created and create new credentials using the
+*'Web Application' Application Type*.
+
+.. image:: google_oauth/web-application.png
+   :align: center
+   :class: img-fluid
+
+Under *Authorized redirect URIs*,  ADD URI: https://yourdbname.odoo.com/google_gmail/confirm
+
+.. note::
+   Replace “yourdbname” in the URL with the Odoo database name.
+
+.. note::
+   `Google’s documentation <https://support.google.com/cloud/answer/6158849#zippy=%2Cnative-applica
+   tions%2Cweb-applications>`__ on Web applications vs. Native applications.
```

Comment:
I think this would better be formatted as a **See Also** tag.
```suggestion
.. seealso::
   `Google’s documentation <https://support.google.com/cloud/answer/6158849#zippy=%2Cnative-applica
   tions%2Cweb-applications>`__ on Web applications vs. Native applications.
```